src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java

```diff
@@ -189,13 +217,22 @@ public void onTraversalStart() {
 
         final VCFHeader inputVCFHeader = getHeaderForVariants();
 
+        if (hasUserSuppliedIntervals()) {
+            if (mergeInputIntervals) {
+                intervals = intervalArgumentCollection.getSpanningIntervals(getBestAvailableSequenceDictionary());
```

Comment:
This should probably be shared code across the two places (here and `initializeIntervals()`). Alternatively you could add a safe getter to `userIntervals` that will let you access these intervals after the merge.